<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtrlVersionModule</class>
 <widget class="QWidget" name="CtrlVersionModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget
{
	background-color:black;
}
QLabel
{
font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
}
</string>
  </property>
  <widget class="QLabel" name="M1Version4lbl">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MP1Version1lbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="M1Version3lbl">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MP2Version4lbl">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>170</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="M2Version3lbl">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="M2Version1lbl">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="M1Version1lbl">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="M1Date1lbl">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;

</string>
   </property>
   <property name="text">
    <string>170816</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="device3lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color :white;
</string>
   </property>
   <property name="text">
    <string>SDB版本</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TC2Version4lbl">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);

</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="M2Version4lbl">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TC2Date1lbl">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>210</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;

</string>
   </property>
   <property name="text">
    <string>170816</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="M2Date1lbl">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;

</string>
   </property>
   <property name="text">
    <string>170816</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TC1Version2lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="M2Version2lbl">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MP1Version2lbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MP2Version3lbl">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="datalbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;

color :white;</string>
   </property>
   <property name="text">
    <string>日期</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TC2Version1lbl">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);

</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TC2Version2lbl">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);

</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MP1Version3lbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TC1Version3lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TC1Version1lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="device1lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color :white;
</string>
   </property>
   <property name="text">
    <string>软件版本</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TC2Version3lbl">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);

</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TC1Version4lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MP2Date1lbl">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>210</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;

</string>
   </property>
   <property name="text">
    <string>170816</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MP2Version1lbl">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="device2lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color :white;
</string>
   </property>
   <property name="text">
    <string>MVB版本</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MP2Version2lbl">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MP1Date1lbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;

</string>
   </property>
   <property name="text">
    <string>170816</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TC1Date1lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;

</string>
   </property>
   <property name="text">
    <string>170816</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="device4lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color :white;
</string>
   </property>
   <property name="text">
    <string>VxWorks版本</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="M1Version2lbl">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MP1Version4lbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="linelbl1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>800</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="titlelbl">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:28px;
color :white;</string>
   </property>
   <property name="text">
    <string>(CCU)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
